### PR TITLE
Create feature: add new reservation - Student C

### PR DESCRIPTION
**Pull request by david-lafontant**
Implement the "add new reservation" form on the Reservations pop up according to the wireframe:
wireframe

### CODE REVIEW

**Comment by david-lafontant** [comment]
Guys, it would be nice, if you could merge this PR if you approved it.

**Comment by denscholar** [comment]
@david-lafontant Please kindly resolve conflicts. this branch has conflicts that need to be resolved before it can be merged with the development branch. Thank you.

**Review by denscholar — CHANGES_REQUESTED** [review]
@david-lafontant You have done a great job. However, it would be nice if you consider the following optional changes as commented. ♻️

**Review by Aduda-Boaz — COMMENTED** [review]
I also have an opinion that we should leave these as default. We will be flagged.

File: .eslintrc.json [hunk 1]
```diff
@@ -13,6 +13,14 @@
...
+    "max-classes-per-file": 0,
+    "no-use-before-define": "off",
+    "no-unused-expressions": "off",
+    "radix": "off",
+    "no-undef": "off",
+    "no-plusplus": "off",
+    "no-unused-vars": "off",
     "eol-last": "off",
     "import/extensions": [ 1, {
       "js": "always", "json": "always"
```
[the remark this answers is not in the document]
Comment (david-lafontant) [reply .eslintrc.json:26]: Asked change implemented

**Review by denscholar — APPROVED** [review]

**Comment by denscholar** [comment]
Thank you for resolving the conflict. I have nothing else other than to merge it to the development branch.

**Pull request merged**